bigtable/client/instance_admin.cc

```diff
@@ -125,9 +125,10 @@ void InstanceAdmin::DeleteInstance(std::string const& instance_id) {
   }
 }
 
-std::vector<btproto::Cluster> InstanceAdmin::ListClusters() {
+std::vector<btproto::Cluster> InstanceAdmin::ListClusters(
```

Comment:
Please add an overload without an `instance_id` parameter that calls the normal implementation, as follows:

```C++
std::vector<btproto::Cluster> InstanceAdmin::ListClusters() {
  return ListClusters("-");
}
```

According to the proto documentation `-` is the magic instance that means "any instance".  We want to support listing all clusters, but do not want the users to know about the magic strings.